templates/vue/src/components/TapestryMain/index.vue

```diff
@@ -100,6 +101,9 @@ export default {
     selectedId: {
       immediate: true,
       handler(nodeId) {
+        setTimeout(() => {
+          client.updateUserLastSelectedNode(nodeId)
+        }, 3000)
```

Comment:
You have this function in two places at the moment which means you are sending 2 requests everytime the node is changed.

Reply:
Removed. Now only calls updateUserLastSelectedNode in TapestryMain